BuildGenerated.sh

```diff
@@ -88,7 +88,7 @@ if [ -z ${SKIPDOWNLOAD+x} ]; then
     echo "Skipping revision/etag-only check of discovery docs"
   else
     # Revert changes that only affect the revision
-    for discovery in $(git status -s -- DiscoveryJson | grep -E '^ M' | cut "-d " -f3)
+    for discovery in $(git status -s -- $DISCOVERY_DOC_DIR | grep -E '^ M' | cut "-d " -f3)
```

Comment:
Can we default to DiscoveryJson so that BuildGenerated.sh still works from the command line without setting anything else?

Reply:
> Basically fine - one suggestion.

And by "fine" I mean "fabulous" :)

Reply:
DISCOVERY_DOC_DIR is not an env var, is a script variable that is set at the top of this same script. Was it being an env var what was the problem?

Reply:
Ah, no - I hadn't realized this was just completing the use of an existing variable. Sorry!